services/IoT/applications/libraries/nodejs.md

```diff
@@ -17,7 +17,7 @@ copyright:
 {: #nodejs}
 
 
-For more information, see:
+For more information, see the following resources:
 - [Client libraries and samples](https://github.com/ibm-messaging/iot-nodejs) in GitHub.
 - [Device samples](https://github.com/ibm-messaging/iot-nodejs/tree/master/samples) in GitHub.
 - [ibmiotf](https://www.npmjs.com/package/ibmiotf) on NPM.
```

Comment:
line 26-29
Application client section
-Not sure why this section is here - does it mean everything underneath it? can some of this be adapated to an overall short description?
-The self-referential sentence should probably be removed or rewritten to focus on why the application client is important. (And is there some other part of node.js that application client is contrasted with? Because right now, I'm having a hard time seeing why this section needs to be here.)